regamedll/dlls/multiplay_gamerules.cpp

```diff
@@ -2066,6 +2070,8 @@ void EXT_FUNC CHalfLifeMultiplay::__API_HOOK(RestartRound)()
 	m_bTargetBombed = m_bBombDefused = false;
 	m_bLevelInitialized = false;
 	m_bCompleteReset = false;
+	
+	FireTargets("game_round_start", nullptr, nullptr, USE_TOGGLE, 0.0);
```

Comment:
Why this without macro?

```suggestion

#ifdef REGAMEDLL_ADD
	FireTargets("game_round_start", nullptr, nullptr, USE_TOGGLE, 0.0);
#endif
```

Reply:
You're right, thank you!